tests/scenarios/compat-route-split-test.ts

```diff
@@ -21,7 +21,9 @@ let splitScenarios = appScenarios.map('compat-splitAtRoutes', app => {
         let app = new EmberApp(defaults, {});
         return maybeEmbroider(app, {
           staticInvokables: true,
-          splitAtRoutes: ['people'],
+          // this is quite complicated because we don't want to match 'people.show' just 'people' to
+          // keep the behaviou to what it was before
+          splitAtRoutes: [/peo([^.\s])+$/],
```

Comment:
I didn't want to create a whole new setup to test the regex so I updated the existing one. It probably makes it overly complicated a test and removes the simple test from our CI 🤔 

Do we think it's worth it writing a new scenario or test setup for just regexes?

Reply:
so my thoughts are that actually we should write a better split route test that covers all the cases we want to cover, this is a quick and dirty smoke test just for serialisation and deserialisation 👍